### ISSUE TITLE
Brave browser

### ISSUE THREAD

**Issue by gavinrogers** [nylira/prism-break#1566]
Based upon many steps that brave boldly makes towards improving security and privacy on a webkit (chromium) foundation, I think it's a good candidate for prism-break. Also, the bitcoin integration (which you can enable in preferences in the newer releases) is a brilliant step forward!

**Comment by strugee** [comment]
@gavinrogers we need way more information than this. For example: a link to the website, a link to the source code, hopefully a link to some documentation on what, exactly, the security and privacy patches do, etc.

**Comment by Atavic** [comment]
No privacy improved on Brave.
See: http://www.ghacks.net/2016/02/25/first-look-at-brave-browser-for-windows/

**Comment by hasufell** [comment]
> According to reports, Brave wants to replace some of the ads on the page with ads of its own in the future. These ads won't track users, and the revenue will be divided between Brave and the user with money paid out via Bitcoin.

Disqualifying, imo. But that's just my opinion.

**Comment by hasufell** [comment]
> can you clarify why?

That's not how I expect a trustworthy browser development team to deal with ads. Making me the asset instead of asking for funding. And therefore I question the goals of that project. That might seem harsh and not particularly technical, but trust isn't just technical.

**Comment by alerque** [comment]
I think this derailed somewhere above because the article quoted is (a) old and (b) not from the source. We're now discussing somebody's commentary on what rumor was last winter.

When Brave first started out they had no revenue model at all and were just blocking all ads. When media asked them about how that would be attractive to publishers they threw out some brainstorm ideas about how it _could_ work down the road and said they were going to explore those ideas.

As best I can figure they did explore that particular idea and rejected it. They actually have a working revenue model now to reimburse publishers that does not involve making users the asset. Their own statement on the affair reads more like this:

> The web has become a different place. With the ad-tech ecosystem out of control, users have revolted and blocking ads has become the new weapon of choice for improving their browsing speed, safety and privacy. Unfortunately, blocking alone results in a race to the bottom where nobody wins. Without the ability for content creators to earn money for their efforts, users could be left with fewer sites to browse, relegated to hand-picked content from controlled sources.
> 
> Brave aims to transform the online ad ecosystem with micropayments and a new revenue-sharing solution to give users and publishers a better deal, where fast, safe browsing is the path to a brighter future for the open web.

In a nutshell Brave users are given the _option_ of paying a fixed amount into a fund (using Bitcoin) and having that divided up anonymously to the owners of sites they frequent.

The speculations from last year left a lot of doubt in my mind too (as per what @hasufell) but I've been watching the company since (and using their browser with an eye to how they are handling this) and I've been impressed with their responsiveness to feedback and evident attention given to privacy ramifications of their model.

I suggest this conversation needs to eschew the article linked above as it is dated, speculative, and even inaccurate — instead we should discuss the current situation which has the model completely flipped from what the article describes.

**Comment by strugee** [comment]
@hasufell makes sense. Thanks for clarifying and generally I agree with your logic.

That being said I for one didn't realise that article was old - thanks @alerque for pointing that out. Someone should dig into their site.

**Comment by Hillside502** [comment]
Brave Software | Building a Better Web
https://brave.com/

Brave Software | About
https://brave.com/about.html

Brave Software | FAQs
https://brave.com/FAQ.html

Introducing Brave Payments
https://blog.brave.com/introducing-brave-payments/

Milestones - brave/browser-laptop
https://github.com/brave/browser-laptop/milestones

Brave Alternatives and Similar Software - AlternativeTo.net
https://alternativeto.net/software/brave/

**Comment by aasami** [comment]
Brave Brovser recently released their version 1.0.0
I thik, this is the time to reconsider what is this browser doing for privacy of its users and whether it is a good candidate for PRISM-break.
[https://brave.com/the-road-to-brave-one-dot-zero/](url)

Brave Launches Next-Generation Browser that Puts Users in Charge of Their Internet Experience with Unmatched Privacy and Rewards
[https://brave.com/brave-launches-next-generation-browser/](url)

Brave, Fingerprinting, and Privacy Budgets
[https://brave.com/brave-fingerprinting-and-privacy-budgets/](url)

**Comment by vyp** [comment]
https://gitlab.com/prism-break/prism-break/issues/1566
